app/controllers/users_controller.rb

```diff
@@ -1,16 +1,33 @@
 class UsersController < ApplicationController
-  before_filter :require_user, only: [:show]
+  before_action :require_user, only: [:show]
   def new
     @user = User.new
   end
 
   def create
     @user = User.new(user_params)
-    if @user.save
-      process_invitation
-      AppMailer.send_welcome_email(@user).deliver
-      redirect_to sign_in_path
+    if @user.valid?
+      token = params[:stripeToken]
+
+      charge = StripeWrapper::Charge.create({
+        amount: 999,
+        currency: "usd",
```

Comment:
Unless you want to be changing currency this could go to the Wrapper as default :).